src/body.js

```diff
@@ -231,6 +232,10 @@ function consumeBody(body) {
  * @return  String
  */
 function convertBody(buffer, headers) {
+	if (typeof convert !== 'function') {
+		throw new Error('The package `encoding` must be installed to use the textConverted() function');
```

Comment:
Should I still be setting a 'type' property here on the Error, even tho it isn't an instance of FetchError? The error document was a little vague on that (It just said 'all errors originating from node-fetch' ...)